tests/functional/contribute/test_friends.py

```diff
@@ -11,9 +11,7 @@
 @pytest.mark.smoke
 @pytest.mark.nondestructive
 def test_signup_default_values(base_url, selenium):
-    page = ContributeFriendsPage(selenium, base_url).open()
-    page.click_show_signup_form()
-    assert page.is_signup_form_displayed
+    page = ContributeFriendsPage(base_url, selenium).open()
```

Comment:
The order of params here should be `selenium, base_url` (this changed just recently when we moved to PyPOM). Same applies to the tests below.